conda_build/utils.py

```diff
@@ -1653,7 +1653,8 @@ def get_logger(name, level=logging.INFO, dedupe=True, add_stdout_stderr_handlers
         logging.config.dictConfig(config_dict)
         level = config_dict.get("loggers", {}).get(name, {}).get("level", level)
     log = logging.getLogger(name)
-    log.setLevel(level)
+    if log.level != level:
+        log.setLevel(level)
     if dedupe:
```

Comment:
I was going to ask that we could also check whether the filter had been already added, but [`logging` itself does that on its own](https://github.com/python/cpython/blob/89bc912dbe277e96e2c7065e06bf8943ddd58eae/Lib/logging/__init__.py#L817-L822), so no need.

Reply:
Yep. I dug into the source code to figure that out too. Fun!